src/helpers/decode.ts

```diff
@@ -0,0 +1,19 @@
+import CryptoJS from 'crypto-js';
+
+export const decryptCode = (email: string, pin: string) => {
```

Comment:
It'd be great to describe this function with JSDoc

Reply:
yee it will be great, we have this from the backend side and I will do the best how I can :)